main.go

```diff
@@ -199,6 +199,13 @@ func main() {
 	}
 	//+kubebuilder:scaffold:builder
```

Comment:
You then need to remove them here. Those tell kubebuilder where to add the next controller and it might add it twice :D